cli/internal/cache/cache_test.go

```diff
@@ -246,9 +246,10 @@ func TestNew(t *testing.T) {
 			wantErr: false,
 		},
 	}
+	var nilClient client
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := New(tt.args.opts, tt.args.config, tt.args.recorder, tt.args.onCacheRemoved)
+			got, err := New(tt.args.opts, tt.args.config, nilClient, tt.args.recorder, tt.args.onCacheRemoved)
```

Comment:
I know it's a bit verbose, but I'd prefer dumping this into the table test so that it isn't a gotcha later.

Reply:
Done.